src/core/lib/security/authorization/sdk_server_authz_filter.cc

```diff
@@ -92,6 +92,18 @@ void SdkServerAuthzFilter::CallData::Destroy(
 bool SdkServerAuthzFilter::CallData::IsAuthorized(SdkServerAuthzFilter* chand) {
   EvaluateArgs args(recv_initial_metadata_batch_,
                     &chand->per_channel_evaluate_args_);
+  if (GRPC_TRACE_FLAG_ENABLED(grpc_sdk_authz_trace)) {
+    gpr_log(
+        GPR_DEBUG,
+        "checking request: url_path:%s, transport_security_type=%s, "
```

Comment:
s/url_path:/url_path=/

Reply:
Missed that. Thank you for the review!:)